### PR TITLE
Prepare 3.23 release

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

File: UPGRADE-3.0.md [hunk 1]
```diff
@@ -1,6 +0,0 @@
-UPGRADE FROM 2.X to 3.0
-=======================
-
-Nothing was documented since now.
-
-If you find BC break and want to contribute, feel free.
```
Comment (greg0ire) [inline UPGRADE-3.0.md:6]: https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1109 should have come with an upgrade note, let's add one now?

Comment (VincentLanglet) [reply UPGRADE-3.0.md:6]: I can, I'll add in 3.x.

I removed the 3.0, since it's pretty old and doesn't add value.

Comment (greg0ire) [reply UPGRADE-3.0.md:6]: You can do it right here since you are targeting 3.x

**Review by greg0ire — APPROVED** [review]

File: UPGRADE-3.0.md (hunk 1 above)
Comment (greg0ire) [reply UPGRADE-3.0.md:6]: Oh right you meant UPGRADE-3.x.md, got it!

**Pull request merged**